core/internal/cluster/kafka_cluster.go

```diff
@@ -98,6 +103,19 @@ func (module *KafkaCluster) Start() error {
 	// Start main loop that has a timer for offset and topic fetches
 	module.offsetTicker = time.NewTicker(time.Duration(module.offsetRefresh) * time.Second)
 	module.metadataTicker = time.NewTicker(time.Duration(module.topicRefresh) * time.Second)
+
+	if module.groupsReaperRefresh != 0 {
+		module.groupsReaperTicker = time.NewTicker(time.Duration(module.groupsReaperRefresh) * time.Second)
+		if !module.saramaConfig.Version.IsAtLeast(sarama.V0_11_0_0) {
```

Comment:
Is there not a variable like `sarama.MIN_VERSION`. I feel we may forget to update this value.

Reply:
we don't need to update this value ever; this only checks that the current client can use the new API added in the protocol v.0.11.0